template/index.js

```diff
@@ -34,14 +34,12 @@ module.exports = yeoman.generators.NamedBase.extend({
         }
     },
 
-    writing: function files() {
+    files: function files() {
         debug("creating template '%s'", this.name);
         this.fs.copyTpl(
             this.templatePath('template.dust'),
             this.destinationPath(path.join('public', 'templates', this.name + '.dust')),
-            {
-                jsModule: this.options.jsModule
-            }
+            this.config.getAll()
```

Comment:
Also we were using `getAll` elsewhere.. I can change it to be explicit in this case if you like. But personally I'd prefer if we just pass along all options always instead. Will go with your call though. Let me know.